src/sesclient.native/sesclient.native.cpp

```diff
@@ -43,62 +49,172 @@ struct Initializer
     }
 };
 
+struct ParameterParser
+{
+public:
+
+    void parse(int argc, char** argv)
```

Comment:
This could return `false` on error instead of setting `_shouldShowUsage` and requiring the caller to check a separate property